src/lib/list/list.ts

```diff
@@ -92,7 +109,7 @@ export class MatListSubheaderCssMatStyler {}
 /** An item within a Material Design list. */
 @Component({
   moduleId: module.id,
-  selector: 'mat-list-item, a[mat-list-item]',
+  selector: 'mat-list-item, a[mat-list-item], button[mat-list-item]',
```

Comment:
We'll need to add documentation for the new list variant to `list.md`

Reply:
done